packages/theme/src/theme/_variables.scss

```diff
@@ -292,6 +292,7 @@ $dropdown-header-color: $gray-light !default;
 $zindex-navbar: 1000 !default;
 $zindex-dropdown: 1000 !default;
 $zindex-navbar-fixed: 1030 !default;
+$zindex-typeahead-items: 1035 !default;
```

Comment:
you can move this one directly in typeahead.scss, this is not part of the theme.

Reply:
done, also deleted defining z-index in multiselect tag https://github.com/Talend/ui/pull/2384/files#diff-d44204ff36a8e26022441648ac114328L56 , so it will be the same as typeahead's zindex